src/main/java/br/com/caelum/vraptor/jpa/extra/ParameterLoader.java

```diff
@@ -95,7 +118,12 @@ public void load(@Observes ControllerFound event) {
 				}
 			}
 		}
-		flash.includeParameters(method, args);
+
+		if (args != null)
+			for (int index = 0; index < args.length; index++)
+				methodInfo.setParameter(index, args[index]);
```

Comment:
When using `flash` the object was setting as null, even though there was a valid object.
Changing to MethodInfo solved this issue.

As I said before, this issue only happens when I change from Observer to Interceptor.
I changed to interceptor to be able to interrupt the stack when no Entity is loaded.